### ISSUE TITLE
"Kubernetes log" can't work without a open yaml file.

### ISSUE THREAD

**Issue by songkaiape** [Azure/vscode-kubernetes-tools#141]
I think this command should behave like other command like "Kubernetes get".
When there is an open yaml file , just get the resource defined in the file. If there is no open yaml file, it should prompt available resource type to choose.

**Comment by kyliel** [comment]
I can see resource type "pod" after I type "Kubernetes: logs". Could you please double confirm with latest version?

**Comment by songkaiape** [comment]
@kyliel  
Minikube version :0.25.2
VScode: 1.22.1
OS: Ubuntu 16.04 &&MacOS
build: 3940 https://vscjavaci.cloudapp.net/job/vscode-k8s-tools.vsix/3940/  &&release build 0.17
I have checked this issue with private build 3940 and release build 0.1.7 .  This issue still repro. 
1. When there is no yaml file open, Executing "Kubernetes log"  will pop out an error "No active editor"
2. When the yaml file is not a pod kind, Executing "Kubernetes log" has no response.

![image](https://user-images.githubusercontent.com/4318793/38598056-71d430de-3d8d-11e8-8a72-2a7b014698a3.png)

**Comment by songkaiape** [comment]
If there is an open file, kubernetes logs will work good. And if you close all editor ,it will throw "No active editor" error

**Comment by itowlson** [comment]
The desired experience for this, per discussion a few weeks back captured in Kylie's email of 6 March, is:

```
Logs: go to terminal. Ask “which object do you want” – same
behavior as “Get”. As for what detailed resource type is supported,
read “kubctl logs”. 
1.	Kubernetes logs
2.	Show drop-down list for pods 
3.	Leave others to terminal commands
```

I don't recall the discussion well enough to fully parse this, but it sounds like we want to ignore any active file and just go straight to the prompt (can we remember why we weren't interested in the active file?).  This would duck the 'no active editor'/non-pod file issue of course.  The available resource types aren't well documented in `kubectl logs` but appear to be at minimum pod, deployment and job (service didn't work for me).

Can anyone remember what 'leave others to terminal commands' referred to?  Are the notes saying we _only_ want to list pods, not deployments or jobs?

We should probably handle the case where more than there is more than one container in the pod, though not sure how easily we can tell this if we support non-pod resource types.

**Comment by itowlson** [comment]
Fixed via #208.
